.github/workflows/clabot.yml

```diff
@@ -6,17 +6,10 @@ on:
     types: [opened,closed,synchronize]
 
 jobs:
-  clabot:
-    runs-on: ubuntu-latest
-    if: contains(github.event.comment.html_url, '/pull/') || github.event_name != 'issue_comment'
-    steps:
-    - name: "CLA Signature Bot"
-      uses: roblox/cla-signature-bot@v2.0.1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        whitelist: "cliffchapmanrbx"
-        use-remote-repo: true
-        remote-repo-name: "roblox/cla-bot-store"
-        remote-repo-pat: ${{ secrets.CLA_REMOTE_REPO_PAT }}
-        url-to-cladocument: "https://roblox.github.io/cla-bot-store/"
+  call-clabot-workflow:
+    uses: Roblox/cla-signature-bot/.github/workflows/clabot-workflow.yml@53aadac6d6f5769d33f955c1611b9fc89f8c9a77
```

Comment:
Is this just for testing? If so should we change the sha reference here to a branch?

Reply:
yep, sha has been changed back to reference version on master